packages/admin-panel-server/src/app/createApp.ts

```diff
@@ -57,6 +62,7 @@ export async function createApp() {
   const builder = new OrchestratorApiBuilder(new TupaiaDatabase(), 'admin-panel')
     .attachApiClientToContext(authHandlerProvider)
     .useSessionModel(AdminPanelSessionModel)
+    .useAttachSession(attachSessionIfAvailable)
```

Comment:
```suggestion
```
I don't think we need this yeah? Since the `requestResetPassword` route is being defined on the `ApiBuilder` and doesn't include the `attachSesssion` middleware.